test/functional/test_1_browser_call.py

```diff
@@ -81,7 +81,7 @@ def switch_to_chatbox(self):
         # for Marionette bug 1094246 to be fixed.
         chatbox = self.wait_for_element_exists(By.TAG_NAME, 'chatbox')
         script = ("return document.getAnonymousElementByAttribute("
-                  "arguments[0], 'class', 'chat-frame');")
+                  "arguments[0], 'anonid', 'content');")
```

Comment:
Half of it, there's a similar change a bit further down. Its also why it is in a separate commit...